src/web/styles.scss

```diff
@@ -99,6 +99,11 @@ ul {
   white-space: pre-wrap;
 }
 
+.tool-tip-decorate:hover {
```

Comment:
`tool-tip-decorate` is a hard-coded class from us, not what is pre-added by Bootstrap. You need to find some other selector if you want to truly target all tooltips.
Also, the effect should not only display on hover (otherwise nobody knows there is a tooltip). What hovering can do, instead, is making the effect more pronounced (e.g. darken the border colour on hover).

Reply:
> Also, the effect should not only display on hover (otherwise nobody knows there is a tooltip). What hovering can do, instead, is making the effect more pronounced (e.g. darken the border colour on hover).

I feel that the user would hover over the element if they don't understand what it is supposed to do or wants additional information. Making the effect show up all the time clutters the UI quite a bit.

![worse-case](https://user-images.githubusercontent.com/32454748/87668612-f78a0800-c79e-11ea-8ee5-e2607351f9c6.png)

Maybe we can make the always present effect more subtle, like italicizing it, and make the underline appear on hover?

![output](https://user-images.githubusercontent.com/32454748/87669356-6ae04980-c7a0-11ea-8fc4-f5cb170fc46c.gif)

Reply:
> Making the effect show up all the time clutters the UI quite a bit.

> like italicizing it, and make the underline appear on hover?

Italic is not more subtle than underline. Also, the underline behaviour is the same as what is achieved via `<abbr>` HTML tag, which also is what most of our tooltips function as.

For that reason, strictly only texts should have the underline decoration in tooltip. And it's not all texts; texts inside button or inside dropdown should also be excluded. Icons should also not have such decoration.

For reference, the same function of the above screenshot from V6:
<img width="886" alt="Screen Shot 2020-07-16 at 8 17 12 PM" src="https://user-images.githubusercontent.com/7261051/87669929-67998d80-c7a1-11ea-8f19-ed891dded271.png">

You can see that there's no underline decoration (to be fair, no tooltip on the text either), in exchange that the information presented becomes rather long.

Reply:
Yes, italics are used for emphasis. We shouldn't use it for a different purpose.

Users shouldn't have to use trial-and-error to discover additional info. It should be obvious where such information is available by eyeballing the UI; shouldn't need to move the mouse around to find out.

Yes, UI clutter is a problem. We can try to make such UI cues subtle so that they don't attract too much attention (e.g., grey dotted underline rather than black dash underline). 

It may be fine to omit the additional visual cue from buttons and icons if most of them have tooltips i.e., the underline doesn't add value in that case because users will know that icons and buttons usually have tooltips. For text with tooltips, most likely we have to stick with underlines as it is common practice elsewhere too.